### PR TITLE
Config

### PR DESCRIPTION
**Pull request by apat2010**
- Work we did on Sunday for basic set up of package.json, wepack.config, and __tests__  to setup a basic jest test

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -41,13 +41,23 @@ class App extends React.Component {
...
         this.setState({users: users.data})
         // console.log('the state is now', this.state)
       })
+      .catch(function(error) {
+        if (!error.status) {
+          // network error
+        }
+      });
 
   }
```
Comment (juanoxmar) [inline client/src/components/App.jsx:63]: Double Checking with you. Thought we were going to do functional react components with hooks. This is a class component.

File: webpack.config.js [hunk 2]
```diff
@@ -10,10 +10,23 @@ module.exports = {
           loader: 'babel-loader', // Allows Babel and Webpack to work together
           options: {
             presets: ['@babel/preset-react', '@babel/preset-env'] // Environment Presets... Not 100% what these do
+          },
+          {
+            test: [/\.css$/],
```
Comment (juanoxmar) [inline webpack.config.js]: not sure if this is what is causing you problems... but i have it written like this: `test: /\.css$/i`

File: webpack.config.js [hunk 3]
```diff
@@ -10,10 +10,23 @@ module.exports = {
           loader: 'babel-loader', // Allows Babel and Webpack to work together
           options: {
```
Comment (juanoxmar) [inline webpack.config.js]: should remove this options property and use a `babel.config.json` file

**Review by juanoxmar — CHANGES_REQUESTED** [review]
Check inline comments

File: client/src/components/App.jsx (hunk 1 above)
Comment (apat2010) [reply client/src/components/App.jsx:63]: Hmmm you are correct. I think I need to spend this afternoon understanding hooks and refactor my existing work before moving on.

File: webpack.config.js (hunk 2 above)
Comment (apat2010) [reply webpack.config.js]: Good call, updated to be correct

File: webpack.config.js (hunk 3 above)
Comment (apat2010) [reply webpack.config.js]: Good call, updated to be correct

**Pull request merged**